modules/workbench/modules/egress_detection/assets/content/query.txt

```diff
@@ -0,0 +1,129 @@
+// This query analyses a set of Google VPC flow logs for high-egress events.
```

Comment:
Is this actually used, or is it intended as a rendered example?

Reply:
the whole query text was downloaded from sumologic UI. I think Greg tried to use this an example to explain this alert?